client.go

```diff
@@ -25,3 +25,40 @@ func (c *client) metrics() (metrics, error) {
 	var res metrics
 	return res, c.rpcClient.Call(&res, "debug_metrics", true)
 }
+
+func (c *client) currentBlock() (string, error) {
+	var res string
+	return res, c.rpcClient.Call(&res, "eth_blockNumber")
+}
+
+func (c *client) currentBlockMetrics() (metrics, error) {
+	/* if not syncing we can at least get current block */
+	cblock, err := c.currentBlock()
+	/* and fake syncing metrics from it */
```

Comment:
Faking metrics does not sound good :D

Reply:
Not my fault `geth` doesn't consider syncing status to be a metric, and I have to do a separate call to get them, and that it changes it's form when syncing is done. It's pretty silly if you ask me.